experimenter/tests/integration/nimbus/conftest.py

```diff
@@ -84,6 +62,30 @@ def slugify(name):
     return name.lower().replace(" ", "-").replace("[", "").replace("]", "")
 
 
+@pytest.fixture(name="application_feature_ids")
```

Comment:
Why this change? I'm not a fan of pytest fixtures tbh, they're pretty magical

Reply:
@brennie That's definitely true but they're a well established pattern in our integration tests and there's no reason to change that now.

Reply:
These are pretty simple fixtures really. This change is because the old dict would get setup once pytest loads the `conftest.py` and for these fenix specific tests, it wasn't needed and causing some failures. This allows us to just call this fixture when we need it.